docs/userGuide/syntax/lists.md

```diff
@@ -42,6 +42,21 @@ You can use any of the [icons](../formattingContents.html#icons) supported by Ma
 If you customize any item on a certain level, you must also customize the first item on that level. If not, the list will revert to its uncustomized form.
 </box>
 
+**You can add an icon that is only for a specific item by using the `i-one-off` attribute.**
```

Comment:
Suggest using `one-off` because after the merging of #2444, this attribute will apply to not only the icon attributes :)

Reply:
`i-once` (to make it shorter)?

Reply:
Would combining the 2 and calling it `once` fine?

Reply:
I think once is ok! @damithc I think not having the `i-` is better because this should apply to the `t-` attributes as well